tracker/dmlc_tracker/local.py

```diff
@@ -43,10 +65,11 @@ def exec_cmd(cmd, role, taskid, pass_env):
             else:
                 raise RuntimeError('Get nonzero return code=%d' % ret)
 
-
 def submit(args):
     """Submit function of local jobs."""
-    def mthread_submit(nworker, nserver, envs):
+    envs = prepare_envs(args)
+
+    def mthread_submit(nworker, nserver, dmlc_envs, addnl_envs):
```

Comment:
can we change the documentation to include dmlc_envs and addnl_envs ?

Reply:
added